### ISSUE TITLE
Missing does not respect explicitly defined locales

### ISSUE THREAD

**Issue by jaredmoody** [glebm/i18n-tasks#588]
When locales are set in the config, I'd expect only missing keys in those locales to be included when `i18n-tasks missing` is run.

Instead, any missing keys in any locales defined in my `data.read` path is returned, and to get the result I want, I have to run: `i18n-tasks missing -l`, passing in all of the locales again.

This seems like a bug to me, but I want to confirm before I submit a PR.

Thanks!

**Comment by davidwessman** [comment]
I'll leave this decision to @glebm

**Comment by glebm** [comment]
Yeah, sounds like a bug. Data should be filtered if the locales are configured explicitly. PR welcome!
